src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  width: 166px;
```

Comment:
Don't repeat yourself. Let's set the width of the card once and remove all of the hardcoded width values from card elements